pkg/operator/certrotation/cabundle.go

```diff
@@ -52,6 +55,8 @@ func (c CABundleRotation) ensureConfigMapCABundle(signingCertKeyPair *crypto.CA)
 		if err != nil {
 			return nil, err
 		}
+		glog.V(2).Infof("Updated ca-bundle.crt configmap %s/%s with:\n%s", certs.CertificateBundleToString(updatedCerts), caBundleConfigMap.Namespace, caBundleConfigMap.Name)
```

Comment:
@deads2k @sttts do you think this is a good spot to print out the bundle? i hope this will allow is to inspect what we had/have in the bundle during e2e runs.

Reply:
make it dependent on the second return value of ApplyConfigMap

Reply:
yes, updated.